### PR TITLE
Refinements to early design review

### PR DESCRIPTION
**Pull request by martinthomson**
Several things here all together.

1. Change the text a little.
2. Use `{{}}` for stuff that is templatey.  The use of `[]` means that we often get design reviews with links called `url`.  Leaving slots blank mean that we have less chance of that.
3. Move parenthetical stuff to the footnotes.
4. Link to chrome status, mozilla, and webkit s-p.

If this makes sense, I'll do the other template in the same way.

### CODE REVIEW

**Review by jyasskin — APPROVED** [review]
This looks like an improvement. The other option is to convert it into a form, which I prototyped in https://github.com/jyasskin/test-design-reviews/issues/7. Upside: it's easy to include explanatory text and placeholders that don't appear in the submitted issue. Downside: validation is still weak, and the result is ugly.

File: .github/ISSUE_TEMPLATE/005-early-design-review.md [hunk 1]
```diff
@@ -10,18 +10,19 @@ assignees: ''
...
-      - [name] ([github username]), [organization/s] (repeat as necessary, we recommend including group chairs and editors in this list)
-  - Organization/project driving the design: [organization and/or project name]
-  - External status/issue trackers for this feature (publicly visible, e.g. Chrome Status):
+  - Explainer¹:
+  - User research:
+  - Security and Privacy self-review²:
+  - GitHub repo:
+  - Primary contacts:
+      - $name (@-mention), $organization/s, $role in developing specification
+      - {{repeat as necessary, we recommend including group chairs and editors in this list}
```
Comment (martinthomson) [inline .github/ISSUE_TEMPLATE/005-early-design-review.md:23]:
```suggestion
      - {{repeat as necessary, we recommend including group chairs and editors in this list}}
```

**Review by torgo — APPROVED** [review]
lgtm

**Review by matatk — APPROVED** [review]
This is ace :-).

**Comment by martinthomson** [comment]
OK, I've made a few more improvements:

* Multi-stakeholder support (for full reviews) or feedback (for early review) is now more consistent between each.
* I've reworked the issue tracking on implementations (for full reviews).
* A note about links rather than pasting content into the issue was added to head off a problem I've observed a few times.

The diff looks pretty big, but it's really just the trimmings.

File: .github/ISSUE_TEMPLATE/010-specification-review.md [hunk 2]
```diff
@@ -10,49 +10,53 @@ assignees: ''
...
+  - Status/issue trackers for implementations⁴:
 
 Further details:
 
   - [ ] I have reviewed the TAG's [Web Platform Design Principles](https://www.w3.org/TR/design-principles/)
-  - Relevant time constraints or deadlines: [please provide]
+  - Relevant time constraints or deadlines:
   - The group where the work on this specification is currently being done:
-  - The group where standardization of this work is intended to be done (if current group is a community group or other incubation venue):
+  - The group where standardization of this work is intended to be done:
```
Comment (jyasskin) [inline .github/ISSUE_TEMPLATE/010-specification-review.md:39]: I think we should keep the note that this won't always be needed. Perhaps:
```suggestion
  - The group where standardization of this work is intended to be done (if different from the current group):
```

File: .github/ISSUE_TEMPLATE/005-early-design-review.md [hunk 3]
```diff
@@ -34,16 +39,19 @@ Further details:
...
+{{Please tell us anything else you think is relevant to this review.}}
 
 ------------------------------------------------------------------------------------
 CAREFULLY READ AND DELETE CONTENT BELOW THIS LINE BEFORE SUBMITTING
 
 Please preview the issue and check that the links work before submitting.
 
-In particular:
-* if anything links to a URL which requires authentication (e.g. Google document), please make sure anyone with the link can access the document. We would prefer public documents though, since we work in the open.
+Use links to content rather than pasting text into this issue.  Issues are ephemeral and most of the material we are asking for has long term value.
```
Comment (jyasskin) [inline .github/ISSUE_TEMPLATE/005-early-design-review.md:49]: I feel like the paragraph before this and the paragraph after should be next to each other. Maybe this one belongs first?

File: .github/ISSUE_TEMPLATE/010-specification-review.md [hunk 4]
```diff
@@ -10,49 +10,53 @@ assignees: ''
...
 
-[please tell us anything you think is relevant to this review]
+{{Please tell us anything else you think is relevant to this review.}}
 
 ------------------------------------------------------------------------------------
 CAREFULLY READ AND DELETE CONTENT BELOW THIS LINE BEFORE SUBMITTING
 
 Please preview the issue and check that the links work before submitting.
 
+Use links to content rather than pasting text into this issue.  Issues are ephemeral and most of the material we are asking for has long term value.
```
Comment (jyasskin) [inline .github/ISSUE_TEMPLATE/010-specification-review.md:52]: I feel like the paragraph before this and the paragraph after should be next to each other. Maybe this one belongs first?

File: .github/ISSUE_TEMPLATE/010-specification-review.md [hunk 5]
```diff
@@ -10,49 +10,53 @@ assignees: ''
...
 
-³ For your own organization, you can simply state the organization's position instead of linking to it. Chromium doesn't have a standards-positions repository and [prefers](https://source.chromium.org/chromium/chromium/src/+/main:docs/standards/positions/GoogleChrome/README.md) to use comments from the teams that maintain the relevant area of their codebase.
+³ For your own organization, you can simply state the organization's position instead of linking to it.  This includes items on [Mozilla standards-positions](https://github.com/mozilla/standards-positions), and [WebKit standards-positions](https://github.com/WebKit/standards-positions).  Chromium doesn't have a standards-positions repository and [prefers](https://source.chromium.org/chromium/chromium/src/+/main:docs/standards/positions/GoogleChrome/README.md) to use comments from the teams that maintain the relevant area of their codebase.
+
+⁴ Include a link to [Chrome Status](https://chromestatus.com/), [Mozilla's](https://bugzilla.mozilla.org/), [WebKit's Bugzilla](https://bugs.webkit.org/), or trackers for other implementations if those are known to you.
```
Comment (jyasskin) [inline .github/ISSUE_TEMPLATE/010-specification-review.md:62]:
```suggestion
⁴ Include links to [Chrome Status](https://chromestatus.com/), [Mozilla's](https://bugzilla.mozilla.org/), [WebKit's Bugzilla](https://bugs.webkit.org/), and/or trackers for other implementations if those are known to you.
```

**Review by jyasskin — APPROVED** [review]
A couple suggestions, but this looks great even if you decide not to take them.

File: .github/ISSUE_TEMPLATE/010-specification-review.md (hunk 5 above)
Comment (martinthomson) [reply .github/ISSUE_TEMPLATE/010-specification-review.md:62]:
```suggestion
⁴ Include links to [Chrome Status](https://chromestatus.com/), [Mozilla's](https://bugzilla.mozilla.org/), [WebKit's Bugzilla](https://bugs.webkit.org/), and trackers for other implementations if those are known to you.
```

A moratorium on "and/or"...

**Pull request merged**